lms/djangoapps/django_comment_client/forum/tests.py

```diff
@@ -532,9 +532,43 @@ def test_group_info_in_ajax_response(self, mock_request):
         )
 
 
+@patch('requests.request')
+class SingleThreadContentGroupTestCase(ContentGroupTestCase):
+    def test_alpha_and_staff_can_see(self, mock_request):
+        thread_id = "test_thread_id"
+        mock_request.side_effect = make_mock_request_impl("dummy content", thread_id)
+        for user_with_access in [self.alpha_user, self.staff_user]:
+            CourseEnrollmentFactory.create(user=user_with_access, course_id=self.course.id)
```

Comment:
I'm confused why this test enrolls the users in the course, but other tests using ContentGroupTestCase do not (I was wondering if the enrollment should happen in ContentGroupTestCase itself).